src/views/product-root-docs-path-landing/server.ts

```diff
@@ -94,6 +95,7 @@ const generateGetStaticProps = ({
     const { getStaticProps: generatedGetStaticProps } =
       _getStaticGenerationFunctions({
         product,
+        productSlugForLoader,
```

Comment:
For HCP docs, the HashiCorp Developer product slug `hcp` does not match the `cloud.hashicorp.com` value we need to send to the content API.